src/util/types.ts

```diff
@@ -98,3 +100,30 @@ export interface OrderBook {
     mySizeOrders: OrderBookItem[];
     spread: BigNumber;
 }
+
+interface BaseNotification {
+    timestamp: Date;
+}
+
+interface CancelOrderNotification extends BaseNotification {
+    kind: 'CancelOrderNotification';
```

Comment:
Non-blocking comment: we can use an enum (or string enum) for the kind attribute.

Reply:
Done, thanks!